### PR TITLE
Added upgrade article for kubevela version 1.2.5

### PR DESCRIPTION
**Pull request by jingliming**
from bizworks

### CODE REVIEW

File: blog/2022-05-27-kubevela-1.2.5-upgrade-practice.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+title: BizWorks' practice on upgrading kubevera 1.2.5
```
Comment (wonderflow) [inline blog/2022-05-27-kubevela-1.2.5-upgrade-practice.md]:
```suggestion
title: BizWorks' practice on upgrading KubeVela from 1.1.5 to 1.2.5
```

File: docs/platform-engineers/system-operation/1.2.5-upgrade-practice.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+---
+title: Upgrade KubeVela from 1.1.5 to 1.2.5
+---
+
```
Comment (wangyikewxgm) [inline docs/platform-engineers/system-operation/1.2.5-upgrade-practice.md:4]: This doc should move to 1.2-versioned docs: https://github.com/kubevela/kubevela.io/tree/main/versioned_docs/version-v1.2/platform-engineers/system-operation

**Review by wangyikewxgm — COMMENTED** [review]
please add a sidebar here https://github.com/kubevela/kubevela.io/blob/61293c8a10e5bd52d778338740291e7fc922f7c9/versioned_sidebars/version-v1.2-sidebars.json#L378 for this doc.

File: versioned_sidebars/version-v1.2-sidebars.json [hunk 3]
```diff
@@ -376,6 +376,10 @@
             {
               "type": "doc",
               "id": "version-v1.2/platform-engineers/system-operation/velaql"
+            },
+            {
+              "type": "doc",
+              "id": "version-v1.2/platform-engineers/system-operation/1.2.5-upgrade-practice"
```
Comment (wonderflow) [inline versioned_sidebars/version-v1.2-sidebars.json]: 高级安装 -> 跨版本迁移 -> 本文档

Comment (jingliming) [reply versioned_sidebars/version-v1.2-sidebars.json]: 具体意思解释下？是需要改标题还是要改文件位置？

Comment (wonderflow) [reply versioned_sidebars/version-v1.2-sidebars.json]: sidebar 位置，放到 高级安装下面

Comment (jingliming) [reply versioned_sidebars/version-v1.2-sidebars.json]: 已修改

**Review by wonderflow — APPROVED** [review]

**Pull request merged**